### DISCUSSION TITLE
DAFoam installation test fail IPOPT

### DISCUSSION THREAD

**Discussion by Ryupuriwatt** [mdolab/dafoam#631] [Installation]
During the installation of DAFoam completing without any errors, the installation test fails specifically for IPOPT. I've attempted to resolve this issue multiple times but still can't determine the exact cause of the problem. 
![image](https://github.com/mdolab/dafoam/assets/115110434/c6268289-d188-4474-91e4-311de3c3a8da)
![image](https://github.com/mdolab/dafoam/assets/115110434/14eb4e4f-9b3c-4301-8c96-50f3fb1504d2)
![image](https://github.com/mdolab/dafoam/assets/115110434/d774e3c1-75f6-46fb-be9f-9741d84ed306)

**Comment by friedenhe** [comment]
Do you see a .so file in your dafoam/packages/miniconda3/python3/packages/miniconda3/lib/python3.8/site-packages/pyoptsparse/pyIPOPT folder?

**Reply by ryupuriwat** [reply]
![659c3c217102a5222df4a3061e8173b](https://github.com/mdolab/dafoam/assets/109540234/e09e4860-068f-4198-8785-86e975eac3cc)
I see this folder but there is no .so file inside this folder. (now I am installing inside HPC tianhe3) but during installation of IPOPT there is no error so could you please give me an suggestion?

**Comment by friedenhe** [comment]
If that is the case, your IPOPT is not properly compiled.. Please double check if you have any error.
